File: ServakApplication/ServakApplication/Models/TimeEventSpan.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ServakApplication.Models
{
    public class TimeEventSpan
    {
        public int Id { get; set; }
        public int UserId { get; set; }
        public DateTime Begin { get; set; }
        public DateTime End { get; set; }
        public string EventName { get; set; }
        public string Color { get; set; }
    }

    public class TimeEventSpanTmp
    {
        public string Begin { get; set; }
        public string End { get; set; }
        public string EventName { get; set; }
        public string Color { get; set; }
    }
}

File: app/src/main/java/the/most/timeapp/models/TimeEventSpan.kt
package the.most.timeapp.models

import java.sql.Time

public class TimeEventSpan {
    public var Id: Int = 0
    public var UserId: Int = 0
    public lateinit var UserName: String
    public lateinit var Begin: String
    public lateinit var End: String
    public lateinit var EventName: String
    public lateinit var Color: String
}